### PR TITLE
Support Python 3.10

### PR DESCRIPTION
**Pull request by lafrech**

### CODE REVIEW

**Comment by lafrech** [comment]
I suggest we run CI on min/max Python versions only. The only benefit would be saving resources. Tests are fast and free, but it just seems like CPU waste. WDYT?

**Comment by sloria** [comment]
Looks like it didn't actually use Python 3.10

Firefox 2021-10-15 at 08 30 24


update: oh, scratch that. python 3.10 does get installed via apt

Firefox 2021-10-15 at 08 29 34

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = lint,py36,py37,py38,py39,docs
+envlist = lint,py36,py37,py38,py39,p310,docs
```
Comment (sloria) [inline tox.ini]:
```suggestion
envlist = lint,py36,py37,py38,py39,py310,docs
```

**Review by sloria — APPROVED** [review]
i'm fine with testing only 3.6 and 3.10. 3.7-3.9 don't have any significant changes that would affect marshmallow, and it'd be nice to reduce our footprint.

File: tox.ini (hunk 1 above)
Comment (lafrech) [reply tox.ini]: 🤦

**Comment by lafrech** [comment]
Good. Thanks for catching the typo.

Next step, [drop Python 3.6 for Christmas](https://endoflife.date/python).

Xmas

**Pull request merged**